python/social_bot/tasks.py

```diff
@@ -982,12 +982,12 @@ def run(self):
                 np.array(obj_pos) - np.array(finger_pos))
             obj_lifted = obj_height / self._obj_init_height - 1.0
             gripping_feature = 0.25 * l_contact + 0.25 * r_contact + min(
-                obj_lifted, 1.0)  # encourge to lift the object by obj_height
-            gripping = (gripping_feature >= 0.999999)
+                obj_lifted, 0.5)  # encourge to lift the object by obj_height
+            gripping = (gripping_feature > 0.99)
```

Comment:
does not need so many 9. 0.99 means gripper have contacts to object and lifted it by 0.49 of object_initial_height. That's enough.